--- a/extern/CMakeProject-pcre.cmake	2022-11-14 22:31:26.000000000 +0000
+++ b/extern/CMakeProject-pcre.cmake	2026-08-14 14:27:17.760123037 +0100
@@ -8,7 +8,7 @@ else()
   source_group("" FILES ${PCRE_SRC})
   source_group("" FILES ${PCRE_HPP})
 
-  add_library("pcre" ${PCRE_SRC} ${PCRE_HPP})
+  add_library("pcre" STATIC ${PCRE_SRC} ${PCRE_HPP})
 
   set_property(TARGET "pcre" PROPERTY FOLDER "External Libraries")
 
